sfdc/rest_client.bal

```diff
@@ -486,8 +424,8 @@ public isolated client class Client {
     # + searchString - Sent SOSL search query
     # + return - `SoslResult` record if successful. Else, the occurred `Error`.
     @display {label: "SOSL Search"}
-    isolated remote function searchSOSLString(@display {label: "SOSL Search Query"} string searchString) returns @tainted@display 
-    {label: "SOSL Result"} SoslResult|Error {
+    isolated remote function searchSOSLString(@display {label: "SOSL Search Query"} string searchString) 
+                                              returns @tainted @display {label: "SOSL Result"} SoslResult|Error {
         string path = prepareQueryUrl([API_BASE_PATH, SEARCH], [Q], [searchString]);
         json res = check self->getRecord(path);
         return toSoslResult(res);
```

Comment:
Don't we need to provide detailed error message for this function?

Reply:
The detailed HTTP error will be returned from inside the `getRecord` function if the error occured in the HTTP layer. Else if a data mapping error occured it will be returned from `toSoslResult` function

Reply:
Ack.